owl/SOMA-IO.owl

```diff
@@ -190,22 +191,12 @@ AnnotationAssertion(rdfs:label SOMA:ArchiveText "Archive Text"@en)
 EquivalentClasses(SOMA:ArchiveText ObjectIntersectionOf(SOMA:Structured_Text ObjectSomeValuesFrom(<http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#expresses> SOMA:FileConfiguration)))
 EquivalentClasses(SOMA:ArchiveText ObjectSomeValuesFrom(SOMA:isGivenMeaningBy SOMA:ArchiveFormat))
 
-# Class: <http://www.ease-crc.org/ont/SOMA.owl#ArtificialAgent> (Artificial agent)
-
-AnnotationAssertion(rdfs:comment SOMA:ArtificialAgent "Any constructed agent with physical embodiment that has intentions on its own, e.g., a robot."@en)
-AnnotationAssertion(rdfs:label SOMA:ArtificialAgent "Artificial agent"@en)
-SubClassOf(SOMA:ArtificialAgent <http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#PhysicalAgent>)
-
-# Class: <http://www.ease-crc.org/ont/SOMA.owl#Classifier> (Classifier)
+# Class: SOMA:Classifier (Classifier)
 
 AnnotationAssertion(rdfs:label SOMA:Classifier "Classifier"@en)
 SubClassOf(SOMA:Classifier SOMA:StatisticalReasoner)
 
-# Class: <http://www.ease-crc.org/ont/SOMA.owl#Client-Server_Specification> (An API Secification that describes the well known Client-Server pattern:
```

Comment:
I approve of removing the commented-out comments as is being done here, since they appear in the annotation property anyway.